### PR TITLE
Topic 1: Maven (new version)

### PR DESCRIPTION
**Pull request by hherminioandres**
-Maven project created.
-The POM file was customized with dependencies: log4j, junit.
-The simple test was created and executed.

### CODE REVIEW

File: target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (carlos-lopez-glb) [inline target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml:1]: The target folder shouldn't be pushed as part of the project in git

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  <dependencies>
+    <dependency>
+    	<groupId>junit</groupId>
+    	<artifactId>junit</artifactId>
+    	<version>3.8.1</version>
+    	<scope>test</scope>
+    </dependency>
+    
+	<dependency>
```
Comment (carlos-lopez-glb) [inline pom.xml]: Please apply formatting to the code

File: src/test/java/com/Maven/java_bootcamp/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.Maven.java_bootcamp;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+public class AppTest
```
Comment (carlos-lopez-glb) [inline src/test/java/com/Maven/java_bootcamp/AppTest.java:7]: is missing the last part of the exercise.

> Then skip the unit test by property or by adding the skipping test configuration to your pom.xml file.

File: pom.xml (hunk 2 above)
Comment (hherminioandres) [reply pom.xml]: Formatting made in pom.xml file.

File: target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml (hunk 1 above)
Comment (hherminioandres) [reply target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml:1]: Unnecessary folder was removed from this project.

File: src/test/java/com/Maven/java_bootcamp/AppTest.java (hunk 3 above)
Comment (hherminioandres) [reply src/test/java/com/Maven/java_bootcamp/AppTest.java:7]: Skipping unit test configuration was added to pom.xml file.

**Review by carlos-lopez-glb — APPROVED** [review]

**Pull request merged**